test/unit/controller/debtor-controller.ts

```diff
@@ -548,4 +548,81 @@ describe('DebtorController', () => {
       expect(res.status).to.equal(400);
     });
   });
+
+  describe('GET /fines/report', () => {
```

Comment:
Maybe also add a test case for your sanity check (that a transfer cannot have a corresponding fine and waivedFine)? Increases test coverage and a higher number is always better.